File: java/javadpbo/src/com/sem4dpbo/week3/praktikum3/latihanpraktikum3_java/Ram.java
package com.sem4dpbo.week3.praktikum3.latihanpraktikum3_java;

public class Ram {
    private String ramCapacity;
    private Long ramPrice;

    //constructor
    Ram(){}
    Ram(String ramCapacity, Long ramPrice){
        this.ramCapacity = ramCapacity;
        this.ramPrice = ramPrice;
    }

    //set and get capacity
    public String getRamCapacity() {
        return ramCapacity;
    }

    public void setRamCapacity(String ramCapacity) {
        this.ramCapacity = ramCapacity;
    }

    //set and get price
    public Long getRamPrice() {
        return ramPrice;
    }

    public void setRamPrice(Long ramPrice) {
        this.ramPrice = ramPrice;
    }

    public void printRam(){
        System.out.println("-Data RAM");
        System.out.println(" Ram Capacity : " + this.ramCapacity);
        System.out.println(" Ram Price : " + this.ramPrice);
        System.out.println("------------------------------------");
    }
}

File: python/3rd_week/latihanpraktikum3_py/Ram.py
class Ram:
    # implementasi class ram

    # atribut
    ramCapacity = ""
    ramPrice = 0

    def __init__(self):
        pass
    
    def setRamCapacity(self, ramCapacity):
        self.ramCapacity = ramCapacity
    
    def getRamCapacity(self):
        return self.ramCapacity

    def setRamPrice(self, ramPrice):
        self.ramPrice = ramPrice
    
    def getRamPrice(self):
        return self.ramPrice

    def printRam(self):
        print("-Data RAM")
        print(f" Kapasitas RAM : {self.ramCapacity}")
        print(f" Harga RAM     : {self.ramPrice}")
        print("--------------------------------------")
    